learn/app-development/custom-widgets/prefab-with-partials.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Prefabs with Partials"
+id: ""
+sidebar_label: "Prefab with Partials"
+---
+---
+
+[Prefabs](/learn/app-development/custom-widgets/prefabs-overview) are reusable application parts that you can integrate into applications easily. To learn how to create Prefabs in WaveMaker, see [Creating Prefabs](/learn/app-development/custom-widgets/creating-prefabs).
+
+With Partials inside a Prefab, you can create higher abstraction levels, and reuse functions across the web and mobile projects. You can use several other features, including organizing UI, using form-factor for enabling different screen size compositions. Additionally, benefit from using features like [Data Table with Row Expansion](/learn/how-tos/how-to-configure-row-expansion-in-a-data-table) and [Page Dialogs](/learn/app-development/widgets/modal-dialogs/modal-windows-dialogs#page-dialog) for on-demand content loading.
+
+## Creating Partials
+
+You can create two types of partials, including **Partials** and **Widget Templates**.
+
+1. **Partials**: Use widgets such as Containers, Popovers, Page Dialogs, and more.  
+
+2. **Widget Templates**: Create [Widget Templates](/learn/how-tos/custom-widget-template) with widgets, including Search, Radioset, Checkboxset to customize the look and feel.
+
+![PartialTypes](/learn/assets/partialTypes.png)
+
+You can import [template bundles](/learn/app-development/ui-design/page-concepts/creating-template-bundles) of Partials and Widget Templates into a Prefab Project. Choose the **Platform Type** when using the **Import Template** option to import for web and mobile projects separately.
+
+## Prefab Properties in Partial
+
+Access the general properties from the Main page by default. To access properties from a Partial, you add them from the **Config Prefab** section. Follow the steps below to achieve this.
+
```

Comment:
This gives an impression that you can add prefab properties only from partials. Can we say something like "Similar to how we can access Prefab properties from Main page, you can access them from Partials as well. To create a property, you need to go the Config Prefab section as shown below."